pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -605,7 +606,7 @@ func (c *Reconciler) reconcile(ctx context.Context, pr *v1beta1.PipelineRun) err
 	before := pr.Status.GetCondition(apis.ConditionSucceeded)
 	after := resources.GetPipelineConditionStatus(pr, pipelineState, c.Logger, d)
 	pr.Status.SetCondition(after)
-	reconciler.EmitEvent(c.Recorder, before, after, pr)
+	events.EmitEvent(c.Recorder, before, after, pr)
```

Comment:
We could even rename `EmitEvent` to `Emit`, would make `event.Emit` :stuck_out_tongue: